src/services/event.test.js

```diff
@@ -59,59 +63,58 @@ describe('event service', () => {
+      const found = await findEventById(created.id);
+      expect(found).toHaveProperty('id');
+      expect(found).toHaveProperty('ownerId', 98_989);
+      await destroyEvent(created.id);
     });
 
-    describe('findEventsByOwnerId', () => {
-      it('works', async () => {
-        const created = await createAnEvent(98_989);
-        const found = await findEventsByOwnerId(created.ownerId);
-        expect(found[0]).toHaveProperty('id');
-        expect(found[0]).toHaveProperty('ownerId', 98_989);
-        await destroyEvent(created.id);
-      });
+    it('findEventsByOwnerId', async () => {
+      const created = await createAnEvent(98_989);
+      const found = await findEventsByOwnerId(created.ownerId);
+      expect(found[0]).toHaveProperty('id');
+      expect(found[0]).toHaveProperty('ownerId', 98_989);
+      await destroyEvent(created.id);
     });
 
-    describe('findEventsByPocId', () => {
-      it('works', async () => {
-        const created = await createAnEvent(98_989);
-        const found = await findEventsByPocId(created.pocId);
-        expect(found[0]).toHaveProperty('id');
-        expect(found[0]).toHaveProperty('ownerId', 98_989);
-        await destroyEvent(created.id);
-      });
+    it('findEventsByPocId', async () => {
+      const created = await createAnEvent(98_989);
+      const found = await findEventsByPocId(created.pocId);
+      expect(found[0]).toHaveProperty('id');
+      expect(found[0]).toHaveProperty('ownerId', 98_989);
+      await destroyEvent(created.id);
     });
 
-    describe('findEventsByCollaboratorId', () => {
-      it('works', async () => {
-        const created = await createAnEvent(98_989);
-        const found = await findEventsByCollaboratorId(created.collaboratorIds[0]);
-        expect(found[0]).toHaveProperty('id');
-        expect(found[0]).toHaveProperty('ownerId', 98_989);
-        await destroyEvent(created.id);
-      });
+    it('findEventsByCollaboratorId', async () => {
+      const created = await createAnEvent(98_989);
+      const found = await findEventsByCollaboratorId(created.collaboratorIds[0]);
+      expect(found[0]).toHaveProperty('id');
+      expect(found[0]).toHaveProperty('ownerId', 98_989);
+      await destroyEvent(created.id);
     });
 
-    describe('findEventsByRegionId', () => {
-      it('works', async () => {
-        const created = await createAnEvent(98_989);
-        const found = await findEventsByRegionId(created.regionId);
-        expect(found[0]).toHaveProperty('id');
-        expect(found[0]).toHaveProperty('ownerId', 98_989);
-        await destroyEvent(created.id);
-      });
+    it('findEventsByRegionId', async () => {
+      const created = await createAnEvent(98_989);
+      const found = await findEventsByRegionId(created.regionId);
+      expect(found[0]).toHaveProperty('id');
+      expect(found[0]).toHaveProperty('ownerId', 98_989);
+      await destroyEvent(created.id);
+    });
+
+    it('findEventsByStatus', async () => {
```

Comment:
Might have missed it, but make sure we have a event that should not come back.

IE another status type.